lazer/sdk/proto/publisher_update.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package pyth_lazer_transaction;
+
+message PublisherUpdate {
+  repeated FeedUpdate updates = 1;
+  uint64 batch_timestamp_us = 2;
+}
+
+message FeedUpdate {
+  oneof update {
+    PriceUpdateV1 price_update_v1 = 1;
```

Comment:
We probably won't need V1 suffix because protobuf has proper schema evolution. If we need it we can rename it later.

Reply:
I recommend seeing how it works internally. I found this one useful: https://martin.kleppmann.com/2012/12/05/schema-evolution-in-avro-protocol-buffers-thrift.html